Source/WebKit/UIProcess/Automation/WebAutomationSession.h

```diff
@@ -279,9 +284,11 @@ class WebAutomationSession final : public API::ObjectImpl<API::Object::Type::Aut
 
     void didDestroyFrame(WebCore::FrameIdentifier);
 
-private:
+protected:
```

Comment:
is this actually necessary since you've declared `friend class WebDriverBidiProcessor;`?